proto/ibc/tendermint/tendermint.proto

```diff
@@ -103,7 +101,6 @@ message Evidence {
 message Header {
   .tendermint.types.SignedHeader signed_header = 1 [
     (gogoproto.embed)    = true,
-    (gogoproto.nullable) = false,
```

Comment:
I realize looking at the diffs that maybe making the signed header a pointer isn't necessary. Thoughts? Should I switch back?

Reply:
if we use a pointer we should add test cases to check for `nil`